enterprise/views.py

```diff
@@ -286,6 +290,12 @@ def get_account_consent(self, request):
             ).format(
                 enterprise_customer_name=customer.name
             ),
+            'confirmation_alert_prompt_warning': _(
+                'If you do not consent to share your course data, that information may be shared with '
```

Comment:
Answering for Tim: this is a clone of the course-specific message. I asked the same question on the original PR. ;)